src/ui/activity-iframe-view.ts

```diff
@@ -69,7 +69,8 @@ export class ActivityIframeView extends View {
     const port = await this.activityPorts_.openIframe(
       this.iframe_,
       this.src_,
-      this.args_
+      this.args_,
+      true
```

Comment:
Since it is not clear from context, can you document the meaning of this argument with an inline comment?